Framework/DataHandling/src/LoadILLDiffraction.cpp

```diff
@@ -95,7 +100,7 @@ void LoadILLDiffraction::exec() {
   m_fileName = getPropertyValue("Filename");
 
   loadScanVars();
-  progress.report("Loaded the scanned variables");
+  progress.report("Loading the scanned variables");
```

Comment:
If you prefer `Loading` please put this above `loadScanVars()`. The same applies to the rest.